cli/tests/integrations/test_config.py

```diff
@@ -12,20 +12,24 @@
 
 @pytest.fixture
 def env():
-    return {
+    r = os.environ.copy()
```

Comment:
Windows requires full copy of the ENV, with next its update. If we put new  poor Env like: {
        constants.PATH_ENV: os.environ[constants.PATH_ENV],
        constants.DCOS_CONFIG_ENV: os.path.join("tests", "data", "dcos.toml"),
        cli_constants.DCOS_PRODUCTION_ENV: 'false'
    } it doesn't work for Windows, and we got next error: 
STDOUT:
STDERR: Traceback (most recent call last):
 File "C:\Users\admin\tmp\dcos-cli\cli\.tox\py27-integration\Scripts\dcos-script.py", line 5, in <module>
   from pkg_resources import load_entry_point
 File "C:\Users\admin\tmp\dcos-cli\cli\.tox\py27-integration\lib\site-packages\pkg_resources\__init__.py", line 37, in <module>
   import email.parser
 File "c:\python27\Lib\email\parser.py", line 12, in <module>
   from email.feedparser import FeedParser
 File "c:\python27\Lib\email\feedparser.py", line 27, in <module>
   from email import message
 File "c:\python27\Lib\email\message.py", line 16, in <module>
   import email.charset
 File "c:\python27\Lib\email\charset.py", line 13, in <module>
   import email.base64mime
 File "c:\python27\Lib\email\base64mime.py", line 40, in <module>
   from email.utils import fix_eols
 File "c:\python27\Lib\email\utils.py", line 27, in <module>
   import random
 File "c:\python27\Lib\random.py", line 885, in <module>
   _inst = Random()
 File "c:\python27\Lib\random.py", line 97, in __init__
   self.seed(x)
 File "c:\python27\Lib\random.py", line 113, in seed
   a = long(_hexlify(_urandom(2500)), 16)
WindowsError: [Error -2146893795] Provider DLL failed to initialize correctly